sources/world/Makefile

```diff
@@ -0,0 +1,103 @@
+
+$(DATA_DIR)/admin_regions_lvl2_v1.geo.json: $(DATA_DIR)/admin_regions_lvl2_v2.topo.json ## Regions TopoJSON
+	mapshaper -i $(DATA_DIR)/admin_regions_lvl2_v2.topo.json \
+  -simplify visvalingam interval=1000 keep-shapes \
+  -sort this.properties.region_iso_code \
+  -o rfc7946 force format=geojson /tmp/admin_regions_lvl2_v1.geo.json
+	node ../../scripts/clean-geom.js -v /tmp/admin_regions_lvl2_v1.geo.json
+	mapshaper -i /tmp/admin_regions_lvl2_v1.geo.json \
+  -o rfc7946 format=geojson prettify $(DATA_DIR)/admin_regions_lvl2_v1.geo.json
+
+/tmp/population.csv: ## Population data from the World Bank API
+	echo "iso,measure" > /tmp/population.csv &&\
+  curl -s "http://api.worldbank.org/v2/country/all/indicator/SP.POP.TOTL?format=json&per_page=300&date=2018" |\
+  jq -cr '.[1][] | select(.countryiso3code != "") | select(.value > 0) | .countryiso3code + "," + (.value | tostring)' |\
+  sort >> /tmp/population.csv
+
+/tmp/area.csv: ## Country area data from the World Bank API
+	echo "iso,measure" > /tmp/area.csv &&\
+  curl -s "http://api.worldbank.org/v2/country/all/indicator/AG.LND.TOTL.K2?format=json&per_page=300&date=2018" |\
+  jq -cr '.[1][] | select(.countryiso3code != "") | select(.value > 0) | .countryiso3code + "," + (.value | tostring)' |\
+  sort >> /tmp/area.csv
+
+/tmp/isos.csv: $(DATA_DIR)/admin_regions_lvl2_v1.geo.json ## Country ISO identifiers from the regions dataset
+	cat $(DATA_DIR)/admin_regions_lvl2_v1.geo.json |\
+  jq -r '.features[] | .properties.country_iso3_code + "," + .properties.country_iso2_code' |\
+  uniq | sort > /tmp/isos.csv
+
+/tmp/missing-area.csv: /tmp/isos.csv /tmp/area.csv ## Missing areas from the World Bank dataset
+	csvsql -H  --query 'select isos.b from isos left join area on isos.a = area.a where area.b is null ' /tmp/isos.csv /tmp/area.csv |\
+  tail +2 > /tmp/missing-area.csv
+
+
+/tmp/missing-population.csv: /tmp/isos.csv /tmp/population.csv ## Missing population from the World Bank dataset
+	csvsql -H  --query 'select isos.b from isos left join population on isos.a = population.a where population.b is null ' /tmp/isos.csv /tmp/population.csv |\
+  tail +2 > /tmp/missing-population.csv
+
+/tmp/wikidata.csv: /tmp/missing-area.csv /tmp/missing-population.csv ## Wikidata area and population for the missing records
+	codes=$$(sort /tmp/missing-area.csv /tmp/missing-population.csv | uniq | awk '{ print "\""$$0"\""}') && curl -H "Accept: text/csv" -sG 'https://query.wikidata.org/sparql' --data-urlencode query="SELECT ?iso2 (MAX(?p) AS ?population) (MAX(?a) AS ?area) WHERE { VALUES ?iso2 { $${codes}  } ?id wdt:P297 ?iso2; wdt:P2046 ?a; wdt:P1082 ?p. } GROUP BY ?iso2 ORDER BY ?iso2" > /tmp/wikidata.csv
+
+$(DATA_DIR)/world_countries_v7.topo.json: $(DATA_DIR)/admin_regions_lvl2_v1.geo.json /tmp/area.csv /tmp/population.csv /tmp/wikidata.csv ## World Countries TopoJSON
+	mapshaper -i $(DATA_DIR)/admin_regions_lvl2_v1.geo.json \
```

Comment:
I think we may have to dissolve from an intermediary admin_regions dataset that does not include the additional regions. Because now we are getting overlapping polygons some countries like French Guiana (as part of FR) and Taiwan (as part of CN). This could be misrepresenting data in our choropleth maps. 

To clarify, I'm OK with adding the additional regions in the admin_regions dataset, but not in the world countries dataset.

Reply:
Maybe I'm being overly cautious. I just tested this layer with a sample dataset. It is possible to page through tooltip results in Elastic Maps. Here's a GIF of what I saw. French Guiana shows up as red because the FR polygon appears first. So the blue polygon for GF does not appear. What do you think?

![Oct-01-2020 13-02-58](https://user-images.githubusercontent.com/1638483/94857533-7bd74900-03e6-11eb-95cf-68659bc844ab.gif)

This might not be the most common case. I imagine China overlapping Taiwan would be more common.

Reply:
This is mostly problematic for Taiwan, and until we have a better way to deal with disputed borders, I think we can stick to our current representation of the world countries.

In a483 I've added a filter to filter the appropriate polygons from the regions dataset before dissolving to the countries. It takes the codes making two SQL queries to the input CSV.